lib/services/screen-builder-service.ts

```diff
@@ -5,18 +5,24 @@ import * as path from "path";
 import * as util from "util";
 
 export class ScreenBuilderService implements IScreenBuilderService {
+	private static SCREEN_BUILDER_SPECIFIC_FILES = [".yo-rc.json", ".app.json", "app.js"];
+	private static UPGRADE_ERROR_MESSAGE = 'Your app has been build with an obsolete version of Screen Builder. Please, migrate it to latest version and retry.';
```

Comment:
It is still unclear whether this error occurs when the project can be automatically migrated or not. We need to figure out if this is a dead-end message or if it gives the user some freedom of action. If it is a dead-end message (meaning that the user should migrate manually), then we should go with "Your app has been created with an obsolete version of Screen Builder. Migrate it manually and retry." If this is a message that says - you can upgrade automatically and need to do it, the message should "Your app has been created with an obsolete version of Screen Builder. Run `$ appbuilder upgrade-screenbuilder` to upgrade it automatically."